python/test/t_GaussianLinearCalibration_chaboche.py

```diff
@@ -0,0 +1,174 @@
+#! /usr/bin/env python
+
+from __future__ import print_function
+import openturns as ot
+from openturns.testing import assert_almost_equal
+
+ot.TESTPREAMBLE()
+ot.PlatformInfo.SetNumericalPrecision(5)
+
+def matrixToSample(matrix):
+    '''Converts a matrix into a Sample, so that we can  
+    assert_almost_equal a Matrix'''
+    size = matrix.getNbRows()
+    dimension = matrix.getNbColumns()
+    sample = ot.Sample(size,dimension)
+    for i in range(size):
+        for j in range(dimension):
+            sample[i,j] = matrix[i,j]
+    return sample
+
+def local2globalCovariance(localErrorCovariance, size, outputDimension):
+    '''
+    Converts a local covariance into a global covariance.
+    size = inputObservations.getSize()
+    outputDimension = modelObservations.getDimension()
+    '''
+    covarianceDimension = size * outputDimension
+    observationDimension = localErrorCovariance.getDimension()
+    globalErrorCovariance = ot.CovarianceMatrix(covarianceDimension)
+    for i in range(size):
+        for j in range(observationDimension):
+            for k in range(observationDimension):
+                globalErrorCovariance[i * observationDimension + j, i * observationDimension + k] = localErrorCovariance[j,k]
+    return globalErrorCovariance
+
+def testCalibrationtChaboche():
```

Comment:
do we really need to define this function ?

Reply:
OK.